packages/calling/src/CallHistory/types.ts

```diff
@@ -41,4 +50,5 @@ export interface ICallHistory extends Eventing<CallHistoryEventTypes> {
     sort: SORT,
     sortBy: SORT_BY
   ): Promise<JanusResponseEvent>;
+  updateMissedCalls(endTimeSessionIds: EndTimeSessionId[]): Promise<UpdateMissedCallsResponse>;
```

Comment:
Please add a description to the `updateMissedCalls` API, similar to getCallHistoryData.